tests/xharness/Harness.cs

```diff
@@ -34,6 +34,11 @@ public class Harness
 			get { return Path.GetFullPath (Path.Combine (RootDirectory, "..", "tools", "xibuild", "xibuild")); }
 		}
 
+		public string DownloadsPath
+		{
+			get { return Path.GetFullPath (Path.Combine (RootDirectory, "..", "builds", "downloads")); }
```

Comment:
You can get the location of the downloaded SDK like this: https://gist.github.com/rolfbjarne/f5108a574f46dd2fcbe639d84c02e3d1

That should simplify some code (and it'll work even if you end up with multiple downloads after mono is bumped).

Reply:
Ah, perfect!

Reply:
Updated after the chat we had in slack.